docs/management/api/blocklist-api.asciidoc

```diff
@@ -0,0 +1,614 @@
+      "updated_by": "elastic"
+    }
+  ],
+  "page": 1,
+  "per_page": 10,
+  "total": 1
+}
+--------------------------------------------------
+
+include::_exceptions-api-reusable-content.asciidoc[tag=endpoint-exceptions-value-types-info-section]
+
+[#exception-{endpoint-artifact-api-doc-name}-item-entry-schema]
+===== `entry` object schema
+
+At least 3 conditions can be defined on an endpoint blocklist entry. Duplicates are not allowed. The following entries are supported by {endpoint-artifact-name}:
+
+====== Process hashes
+
+Process hashes are supported by all xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[OS types]. A hash entry has the following structure:
+
+* `field` : The `process.hash.md5`, `process.hash.sha1` or `process.hash.sha256` values are supported.
+* `value` : An array of hash values associated with the `field`.
+* `type` : Must be `match_any`.
+* `operator` : Must be `included`.
+
+[source,json]
+-----------------------------
+{
+    "field": "file.hash.md5",
+    "value": [
+        "741462ab431a22233C787BAAB9B653C7"
+    ],
+    "type": "match_any",
+    "operator": "included"
+}
+-----------------------------
+
+
+====== Process file path
+
+Process file paths are supported by all xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[OS types]. A file path entry has the following structure:
+
+* `field` : Must be `file.path`.
+* `value` : An array of file paths to match on.
+* `type` : Must be `match_any`.
+* `operator` : Must be `included`.
+
+[source,json]
+-----------------------------
+{
+  "field": "file.path",
+  "value": "c:/path/to/file.exe",
+  "type": "match_any",
+  "operator": "included"
+}
+-----------------------------
+
+
+====== Process signature
+
+Process signature is supported for Windows xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[OS type] only. A signature entry has the following structure:
+
+* `field` : Must be set to `process.Ext.code_signature`.
+* `type` : Must be set to `nested`.
+* `entries`: An array with 1 `entry` item.
+* `entries[0]` : The entry defining the signature to be matched upon:
+** `field`: Must be set to `subject_name`.
+** `value` : An array of signature names to match on.
+** `type` : Must be set to `match_any`.
+** `operator` : Must be set to `included`.
+
+[source,json]
+-----------------------------
+{
+    "field": "file.Ext.code_signature",
+    "entries": [
+        {
+            "field": "subject_name",
+            "value": [
+                "test1",
+                "test2"
+            ],
+            "type": "match_any",
+            "operator": "included"
+        }
+    ],
+    "type": "nested"
+}
+-----------------------------
+
+
+====== Examples for a Windows {endpoint-artifact-name-single}. 
+
+Only one entry item is allowed per blocklist item (`file.path`, `file.hash.*` or `file.Ext.code_signature`). The exception is `file.hash.*`, which allows an entry for each of the allowed hash types.
```

Comment:
Is the comma after `sha1,` needed? Also, there is a missing empty space after the `md5,` right? Updating it will allow me to commit your suggestions, thanks 🙂

Reply:
updated the missing space after md5. Yeah, the comma is needed after sha1.

Reply:
Would like to know @nastasha-solomon thoughts on this as she did also a review on this specific sentence 🙂

Reply:
Thanks for the check @ashokaditya and for the tag @dasansol92 ! I worked with @benironside to make some minor adjustments. Hopefully they're helpful!

```suggestion
Only one entry of `file.path`, `file.hash.*` or `file.Ext.code_signature` is allowed per blocklist item. Each entry can have one or more values. You can also add a list of hashes for each allowed hash type (`md5`, `sha1`, or `sha256`) to the `file.hash.*` entry.
```